src/sass/layout/_footer.scss

```diff
@@ -0,0 +1,127 @@
+.footer-section {
+  background-color: #fff5f6;
+}
+
+.footer{
+  margin-bottom: 43px;
+
+  .social-media {
+    margin-left: auto;
+    margin-bottom: 46px;
+  }
+}
+
+.footer__title--container {
+  max-width: 234px;
+  margin-bottom: 43px;
+}
+
+.footer__superscription {
+  margin-bottom: 17px;
+  text-align: left;
+}
+
+.footer__title {
+  text-align: left; 
+}
+
+.footer__address {
+  font-style: normal;
+  font-weight: $fontBold;
+  line-height: 1.222;
+  text-align: right;
+}
+
+.footer__tel-link {
+  color: $mainColor;
+  text-decoration-line: none;
+}
+
+.footer__copyright--container {
+  border-top: 1px solid #E1E1E1;
+  padding-top: 21px;
+  padding-bottom: 25px;
+}
+
+.footer__copyright {
+  max-width: 149px;
+  margin: 0 auto;
+  font-weight: $fontBold;
+  font-size: $fontSizeSecondary;
+  line-height: 1.5;
+  text-align: center;
+  color: $colorFont2;
+}
+
+@media screen and (min-width: 768px) {
+  .footer{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 91px;
+  
+    .social-media {
+      margin-left: auto;
+      margin-bottom: 46px;
+    }
+  }
+  
+  .footer__title--container {
+    max-width: 340px;
+    margin-bottom: 0;
+    padding-top: 7px;
+  }
+  
+  .footer__superscription {
+    margin-bottom: 16px;
+    letter-spacing: 0.04em;
+  }
+  
+  .footer__title {
+    letter-spacing: 0.06em;
+  }
+  
+  .footer__address {
+    line-height: 1.857;
+  }
+  
+  .footer__copyright--container {
+    padding-top: 28px;
+    padding-bottom: 18px;
+  }
+  
+  .footer__copyright {
+    max-width: none;
+    line-height: 1.857;
+    letter-spacing: 0.02em;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .footer{
+    margin-bottom: 98px;
+  
+    .social-media {
+      margin-bottom: 50px;
+      padding-right: 6px;
+    }
+  }
+  
+  .footer__title--container {
+    max-width: 474px;
```

Comment:
Napis łamie się, powiększyć minimum do 477px

Reply:
Dzięki, faktycznie przeoczyłem to. Dawałem taki jak w layoucie i jakoś nie zauważyłem :)